configs/milvus.yaml

```diff
@@ -367,7 +367,7 @@ queryCoord:
   channelExclusiveNodeFactor: 4 # the least node number for enable channel's exclusive mode
   collectionObserverInterval: 200 # the interval of collection observer
   checkExecutedFlagInterval: 100 # the interval of check executed flag to force to pull dist
-  updateCollectionLoadStatusInterval: 5 # 5m, max interval for updating collection loaded status
+  updateCollectionLoadStatusInterval: 300 # 300s, max interval of updating collection loaded status for check health
```

Comment:
please don't change the unit of the config since this might casue serious compatible issues